lib/digest.js

```diff
@@ -11,7 +11,8 @@ function key(axiosConfig) {
     method: axiosConfig.method,
     data: axiosConfig.data,
     headers: _.omit(axiosConfig.headers, [
-      'Content-Length', 'content-length'
+      'Content-Length', 'content-length',
+      'User-Agent', 'user-agent'
```

Comment:
do you know if axios stopped injecting its own user-agent header?
that was one of the reasons I needed some of those workarounds.

Reply:
@nettofarah this change was made quite some time ago: https://github.com/axios/axios/commit/db85c7bf3ae19d680f5c16cd85d06c5e11fedc5f - Axios only injects its own `User-Agent` header if none is set.

Reply:
@nettofarah I obviously can't read! Your original comment stated the exact same thing 😄  

Removing the `User-Agent` is an alternate strategy that I decided to pursue after testing React components with axios-vcr + Axios + jsdom. In order to simulate a browser environment, jsdom actually provides an implementation of [`XmlHttpRequest`](https://github.com/tmpvar/jsdom/blob/master/lib/jsdom/living/xmlhttprequest.js), which causes axios to use the xhr adapter.

Thanks to [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS), the addition of a custom `User-Agent` means that the request is no longer classified as "simple" but rather one that needs to preflighted. As such, an extra `OPTIONS` request is fired before the actual request (and in my particular use case, this is something our server doesn't support).

If we decide to reinstate the code injecting the `User-Agent: axios-vcr` header, we would first need to check if the original adapter is http.

Reply:
> If we decide to reinstate the code injecting the User-Agent: axios-vcr header, we would first need to check if the original adapter is http.

If I understand things correctly, this might be the safer option here.